core/domain/user_services_test.py

```diff
@@ -863,6 +1122,32 @@ def test_removing_default_user_role_raises_exception(self):
         with self.assertRaisesRegex(Exception, error_msg):
             user_services.remove_user_role(user_id, feconf.ROLE_ID_FULL_USER)
 
+    def test_is_user_registered(self):
+        auth_id = 'test_id'
+        user_email = 'test@email.com'
+        user_id = user_services.create_new_user(auth_id, user_email).user_id
+
+        self.assertTrue(user_services.is_user_registered(user_id))
+
+    def test_is_user_registered_for_non_user_id_returns_false(self):
+        user_id = None
+        self.assertFalse(user_services.is_user_registered(user_id))
+
+    def test_has_fully_registered_account(self):
```

Comment:
Ditto.

Reply:
I don't know how to make the function name state the test more explicitly without making the name too long (< 80 cols). I will be able to do so if I am allowed to abbreviate some terms (e.g. account --> acc). Is that allowed?

Reply:
No, please don't abbreviate.

What are you trying to test here? Perhaps something like test_has_fully_registered_account_for_properly_registered_user()?

Reply:
Done